server/services/bodyScanInfoDecorator.ts

```diff
@@ -4,19 +4,32 @@ import { associateBy } from '../utils/utils'
 
 type HasPrisonNumber = { prisonNumber: string }
 
-export type WithBodyScanInfo<T extends HasPrisonNumber> = T & { bodyScanStatus: BodyScanStatus }
+export type WithBodyScanStatus<T extends HasPrisonNumber> = T & { bodyScanStatus: BodyScanStatus }
+
+export type WithBodyScanInfo<T extends HasPrisonNumber> = T & {
+  numberOfBodyScans: number
+  numberOfBodyScansRemaining: number
+}
 
 export class BodyScanInfoDecorator {
   constructor(
     private readonly hmppsAuthClient: HmppsAuthClient,
     private readonly bodyScanClientFactory: RestClientBuilder<BodyScanClient>
   ) {}
 
-  public async decorate<T extends HasPrisonNumber>(items: T[]): Promise<WithBodyScanInfo<T>[]> {
+  public async decorate<T extends HasPrisonNumber>(items: T[]): Promise<WithBodyScanStatus<T>[]> {
     const token = await this.hmppsAuthClient.getSystemClientToken()
     const prisonNumbers = items.map(i => i.prisonNumber).filter(Boolean)
     const scanInfo = await this.bodyScanClientFactory(token).getBodyScanInfo(prisonNumbers)
     const prisonNumberToScan = associateBy(scanInfo, info => info.prisonNumber)
     return items.map(i => ({ ...i, bodyScanStatus: prisonNumberToScan.get(i.prisonNumber)?.bodyScanStatus }))
   }
```

Comment:
Thought it best to keep current body scan decorator as is to update with `bodyScanStatus` only and not introduce additional body scan data to the recent arrival list page or elsewhere where we only need to know if the arrival can be scanned or not by referencing their `bodyScanStatus`